services/workflows-service/scripts/workflows/dynamic-ui-workflow.ts

```diff
@@ -408,6 +408,7 @@ export const dynamicUiWorkflowDefinition = {
         deliverEvent: 'KYC_REVISION',
       },
     ],
+    workflowLevelResolution: true,
```

Comment:
Does this need to be reverted?

Reply:
Nope, missing resolution level flag causing requests to fail due to missing postUpdateEvents in definition